libs/sdk-ui-kit/src/Icon/icons/Book.tsx

```diff
@@ -0,0 +1,34 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+
+import { IIconProps } from "../typings";
+
+/**
+ * @internal
+ */
+export const Book: React.FC<IIconProps> = ({ color, className, width, height }) => {
+    return (
+        <svg
+            className={className}
+            width={width ?? 16}
+            height={height ?? 16}
```

Comment:
Should this be 11? (according to viewBox value)

Reply:
yes, it should. I will fix it